api-gateway/src/test/java/com/petclinic/bffapigateway/presentationlayer/ApiGatewayControllerTest.java

```diff
@@ -206,38 +243,74 @@ void deleteUser() {
         assertEquals(null, authServiceClient.getUser(user.getId()));
     }
 
-//    @Test
-//    void getPutRequestOk()  {
-//
-//        OwnerDetails od = new OwnerDetails();
-//        od.setId(1);
-//        od.setFirstName("John");
-//        od.setLastName("Doe");
-//        od.setAddress("1 Star Street");
-//        od.setCity("Boston");
-//        od.setTelephone("5553451125");
-//
-//        Mockito.when(customersServiceClient.updateOwner(od,1))
-//                .thenReturn(Mono.just(od));
-//
-//        client.put()
-//                .uri("api/gateway/owners/{ownerId}", 1)
-//                .contentType(MediaType.APPLICATION_JSON)
-//                .accept(MediaType.APPLICATION_JSON)
-//                .exchange()
-//                .expectStatus().isOk()
-//                .expectBody();
-//
-//
-//        assertEquals(od.getId(), 1);
-//        assertEquals(od.getFirstName(), "John");
-//        assertEquals(od.getLastName(),"Doe");
-//        assertEquals(od.getAddress(), "1 Star Street");
-//        assertEquals(od.getCity(), "Boston");
-//        assertEquals(od.getTelephone(),"5553451125");
-//
-//
-//    }
+
+
+    //private static final int BILL_ID = 1;
+
+    @Test
+    public void getBillById(){
```

Comment:
Remember Jason's naming convention for tests it should start with "should" or "test" or something like that. Applies to all tests.